codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/presignable/PresignableGetIntegration.kt

```diff
@@ -166,7 +178,15 @@ class PollyGetPresignerIntegration(private val presignedOperations: Map<String,
             .build()
         delegator.useShapeWriter(headerMiddlewareSymbol) { writer ->
             writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)
-            val queryItemMiddleware = PollySynthesizeSpeechGETQueryItemMiddleware(ctx, inputSymbol, outputSymbol, outputErrorSymbol, inputShape, writer)
+            val queryItemMiddleware =
+                software.amazon.smithy.aws.swift.codegen.customization.InputTypeGETQueryItemMiddleware(
```

Comment:
Nit: Can we add the import here to the top of the file instead of having this inline?

Reply:
are you referring to the line that says: `writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)` -- where else would we put this?

Reply:
OH! i see.. wow.. i wonder why it didn't add an import statement.